server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const path = require('path');
+const app = express();
+const PORT = 3000;
+
+app.use(express.static(path.join(__dirname + '/dist')));
+
+app.listen(PORT, function () {
```

Comment:
В таки конструкциях обычно используют стрелочные функции:

```suggestion
app.listen(PORT, () => {
```